debian/rules

```diff
@@ -16,5 +16,14 @@ BUILDDIR=build_dir
 export DEB_CFLAGS_MAINT_APPEND   = -Wno-stringop-overflow -Wno-maybe-uninitialized -Wno-error=null-dereference
 export DEB_CXXFLAGS_MAINT_APPEND = -Wno-stringop-overflow -Wno-maybe-uninitialized -Wno-error=null-dereference
 
+# https://manpages.debian.org/testing/dpkg-dev/dpkg-architecture.1.en.html#Usage_in_debian/rules
+DEB_BUILD_GNU_TYPE ?= $(shell dpkg-architecture -qDEB_BUILD_GNU_TYPE)
+DEB_HOST_GNU_TYPE ?= $(shell dpkg-architecture -qDEB_HOST_GNU_TYPE)
+
+# Set the number of jobs for armhf to avoid OOMing on older GCC versions (affects builds for Ubuntu 22.04)
```

Comment:
We don't support 22 anymore (https://github.com/OpenRCT2/OpenRCT2/pull/22493) but this is still present on 24

```suggestion
# Set the number of jobs for armhf to avoid OOMing on older GCC versions (affects builds for Ubuntu 24.04)
```

Reply:
On our CI, we don’t support it, but the PPA still does.
```suggestion
# Set the number of jobs for armhf to avoid OOMing on older GCC versions (affects builds for Ubuntu 22.04 and 24.04)
```

Reply:
It does not:
![image](https://github.com/user-attachments/assets/ae6433c8-5de8-4713-9275-d375ff15a35f)

Reply:
My bad.